Makefile

```diff
@@ -129,9 +154,29 @@ cluster/clean:
 	- kubectl delete -f deploy/role.yaml
 	- kubectl delete -n $(NAMESPACE) -f deploy/role_binding.yaml
 	- kubectl delete -n $(NAMESPACE) -f deploy/service_account.yaml
-	- kubectl delete -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_pushapplication_crd.yaml
-	- kubectl delete -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_androidvariant_crd.yaml
-	- kubectl delete -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_iosvariant_crd.yaml
-	- kubectl delete -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml
+	- kubectl delete -f deploy/crds/push_v1alpha1_pushapplication_crd.yaml
+	- kubectl delete -f deploy/crds/push_v1alpha1_androidvariant_crd.yaml
+	- kubectl delete -f deploy/crds/push_v1alpha1_iosvariant_crd.yaml
+	- kubectl delete -f deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml
 	- make monitoring/uninstall
-	- kubectl delete namespace $(NAMESPACE)
\ No newline at end of file
+	- kubectl delete namespace $(NAMESPACE)
+	- kubectl delete namespace ${APP_NAMESPACES}
+
+
+.PHONY: image/build/master
+image/build/master:
+	operator-sdk build quay.io/${ORG_NAME}/${APP_NAME}:master
+
+ .PHONY: image/push/master
```

Comment:
I'd remove these tasks to build & push master. The Jenkins pipeline does that already, and we don't want people to think they can overwrite it with an image they built on their own machine.

Reply:
👍